tasks/main.yml

```diff
@@ -50,6 +59,14 @@
   when: "{{ freshclam_enabled }}"
   tags: usb
 
+- name: Set cron job for scanning / at 12:00 on a Wednesday
```

Comment:
All the ClamAV documentation I found suggests adding the job to be run as a cron job